src/gui/AccountConfigTab.py

```
@@ -74,30 +74,31 @@ def has_user_config(self):
 
 class AccountConfigTab(CustomTab):
     """Tab for managing per-account configuration overrides and account list."""
+
     ALWAYS_HIDDEN_CONFIG_KEYS = {"多账户模式", "多账户独立配置", "账号列表"}
```

Ruff (0.16.2)

[warning] Mutable default value for class attribute

(RUF012)